src/createServer.js

```diff
@@ -4,0 +1,78 @@
+const http = require('http');
+const { detectCase } = require('./convertToCase/detectCase');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const allCases = ['SNAKE', 'KEBAB', 'UPPER', 'CAMEL', 'PASCAL'];
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const errors = [];
+
+    try {
+      const [path, queryString] = req.url.split('?');
+      const params = new URLSearchParams(queryString);
```

Comment:
You are splitting the URL and manually parsing the query string. Consider using a library like 'url' or 'query-string' to parse the URL and query parameters. It can simplify your code and make it more robust.